### ISSUE TITLE
Problems with importlib.metadata

### ISSUE THREAD

**Issue by tobeycarman** [pyproj4/pyproj#1177]
I have a bokeh application that uses `pyproj` and the application is failing to run in certain circumstances due to an error coming from `pyproj`, specifically the `importlib.metadata` use in `__init__.py`.

I originally posted this in the Bokeh community discussion where Bryan figured out that the issue was actually coming from `pyproj`. See full discussion here: 

https://discourse.bokeh.org/t/bokeh-server-with-geopandas-pyproj/9722/3
Also there is another question posted to the general Python Discuss here:
https://discuss.python.org/t/module-importlib-has-no-attribute-metadata/20947

#### Platform

On a compute note of an HPC cluster running CentOS v7.x I think. I have been using Conda to manage packages in my home directory.

Setup like so:
```
conda create -n py38 python=3.8
conda install gdal    # for libgdal
pip install gdal      # for python bindings
python -c "from osgeo import gdal" # works
pip install ipython pandas geopandas fiona rasterio
pip install netCDF4 configobj
```

#### Requirements.txt  

<details>

```
affine==2.3.1
anyio==3.6.2
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asttokens==2.1.0
attrs==22.1.0
backcall==0.2.0
beautifulsoup4==4.11.1
bleach==5.0.1
bokeh==3.0.1
certifi==2022.9.24
cffi==1.15.1
cftime==1.6.2
click==8.1.3
click-plugins==1.1.1
cligj==0.7.2
configobj==5.0.6
contourpy==1.0.6
coverage==5.5
debugpy==1.6.3
decorator==5.1.1
defusedxml==0.7.1
deprecation==2.1.0
entrypoints==0.4
executing==1.2.0
eyed3==0.9.7
fastjsonschema==2.16.2
filetype==1.2.0
Fiona==1.8.22
GDAL==3.5.3
geojson==2.5.0
geopandas==0.12.1
idna==3.4
importlib-resources==5.10.0
ipykernel==6.17.1
ipython==8.6.0
ipython-genutils==0.2.0
ipywidgets==8.0.2
jedi==0.18.1
Jinja2==3.1.2
jsonschema==4.17.0
jupyter==1.0.0
jupyter-console==6.4.4
jupyter-server==1.23.1
jupyter_client==7.4.5
jupyter_core==5.0.0
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.3
MarkupSafe==2.1.1
matplotlib-inline==0.1.6
mistune==2.0.4
munch==2.5.0
nbclassic==0.4.8
nbclient==0.7.0
nbconvert==7.2.4
nbformat==5.7.0
nest-asyncio==1.5.6
netCDF4==1.6.1
notebook==6.5.2
notebook_shim==0.2.2
numpy @ file:///home/conda/feedstock_root/build_artifacts/numpy_1666788162538/work
packaging==21.3
pandas==1.5.1
pandocfilters==1.5.0
parso==0.8.3
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.3.0
pkgutil_resolve_name==1.3.10
platformdirs==2.5.3
prometheus-client==0.15.0
prompt-toolkit==3.0.32
psutil==5.9.4
ptyprocess==0.7.0
pure-eval==0.2.2
pycparser==2.21
pyglet==1.1.4
Pygments==2.13.0
pyparsing==3.0.9
pyproj==3.4.0
pyrsistent==0.19.2
python-dateutil==2.8.2
pytz==2022.6
PyYAML==6.0
pyzmq==24.0.1
qtconsole==5.4.0
QtPy==2.3.0
rasterio==1.3.3
Send2Trash==1.8.0
Shapely==1.8.5.post1
six==1.16.0
sniffio==1.3.0
snuggs==1.4.7
soupsieve==2.3.2.post1
stack-data==0.6.0
terminado==0.17.0
tinycss2==1.2.1
toml==0.10.2
tornado==6.2
traitlets==5.5.0
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.4.2
widgetsnbextension==4.0.3
xyzservices==2022.9.0
zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1666647772197/work
```
</details>

#### Run with

```bash
$ bokeh serve --port 8686 MRE_io_view.py 
2022-11-10 11:47:43,312 Starting Bokeh server version 3.0.1 (running on Tornado 6.2)
2022-11-10 11:47:44,091 User authentication hooks NOT provided (default user enabled)
2022-11-10 11:47:44,102 Bokeh app running at: http://localhost:8686/MRE_io_view
2022-11-10 11:47:44,102 Starting Bokeh server with process id: 10551
BokehDeprecationWarning: tile_providers module was deprecated in Bokeh 3.0.0 and will be removed, use add_tile directly instead.
2022-11-10 11:47:48,508 Error running application handler <bokeh.application.handlers.script.ScriptHandler object at 0x7f47b76e3070>: module 'importlib' has no attribute 'metadata'
File '__init__.py', line 70, in <module>:
__version__ = importlib.metadata.version(__package__) Traceback (most recent call last):
  File "/home/UA/tcarman2/miniconda3/envs/py38/lib/python3.8/site-packages/bokeh/application/handlers/code_runner.py", line 231, in run
    exec(self._code, module.__dict__)
  File "/home/UA/tcarman2/dvm-dos-tem/scripts/MRE_io_view.py", line 28, in <module>
    import geopandas
  File "/home/UA/tcarman2/miniconda3/envs/py38/lib/python3.8/site-packages/geopandas/__init__.py", line 1, in <module>
    from geopandas._config import options  # noqa
  File "/home/UA/tcarman2/miniconda3/envs/py38/lib/python3.8/site-packages/geopandas/_config.py", line 109, in <module>
    default_value=_default_use_pygeos(),
  File "/home/UA/tcarman2/miniconda3/envs/py38/lib/python3.8/site-packages/geopandas/_config.py", line 95, in _default_use_pygeos
    import geopandas._compat as compat
  File "/home/UA/tcarman2/miniconda3/envs/py38/lib/python3.8/site-packages/geopandas/_compat.py", line 9, in <module>
    import pyproj
  File "/home/UA/tcarman2/miniconda3/envs/py38/lib/python3.8/site-packages/pyproj/__init__.py", line 70, in <module>
    __version__ = importlib.metadata.version(__package__)
AttributeError: module 'importlib' has no attribute 'metadata'
 
2022-11-10 11:47:48,837 WebSocket connection opened
2022-11-10 11:47:48,838 ServerConnection created
```

#### Code

```python
#!/usr/bin/env python

# Minimum Reproducible Example for 
# Interactive viewer for mapping collections of dvm-dos-tem input datasets.

import os

from osgeo import gdal
from osgeo import osr

import shapely

import geojson
import json
import glob

from bokeh.io import output_file, show, curdoc, save
from bokeh.models.sources import GeoJSONDataSource
from bokeh.models import HoverTool, TapTool, Patches
from bokeh.models.callbacks import CustomJS
from bokeh.plotting import figure

from bokeh.tile_providers import get_provider, Vendors

import geopandas

print("I am here.")

p = figure()
curdoc().add_root(p)
```

#### Environment Information
```bash
$ pyproj -v
pyproj info:
    pyproj: 3.4.0
      PROJ: 9.1.0
  data dir: /home/UA/tcarman2/miniconda3/envs/py38/lib/python3.8/site-packages/pyproj/proj_dir/share/proj
user_data_dir: /home/UA/tcarman2/.local/share/proj
PROJ DATA (recommended version): 1.11
PROJ Database: 1.2
EPSG Database: v10.074 [2022-08-01]
ESRI Database: ArcGIS Pro 3.0 [2022-07-09]
IGNF Database: 3.1.0 [2019-05-24]

System:
    python: 3.8.13 | packaged by conda-forge | (default, Mar 25 2022, 06:04:18)  [GCC 10.3.0]
executable: /home/UA/tcarman2/miniconda3/envs/py38/bin/python3.8
   machine: Linux-3.10.0-1160.31.1.el7.x86_64-x86_64-with-glibc2.10

Python deps:
   certifi: 2022.9.24
    Cython: None
setuptools: 65.5.1
       pip: 22.3.1
```

 - Python version `python -c "import sys; print(sys.version.replace('\n', ' '))"` --> 3.8.13 | packaged by conda-forge | (default, Mar 25 2022, 06:04:18)  [GCC 10.3.0]
 - Operation System Information `python -c "import platform; print(platform.platform())"` --> Linux-3.10.0-1160.31.1.el7.x86_64-x86_64-with-glibc2.10

#### Conda environment information (if you installed with conda):

<br/>
Environment (<code>conda list</code>):
<details>
```
$ conda list proj
# packages in environment at /home/UA/tcarman2/miniconda3/envs/py38:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
affine                    2.3.1                    pypi_0    pypi
anyio                     3.6.2                    pypi_0    pypi
argon2-cffi               21.3.0                   pypi_0    pypi
argon2-cffi-bindings      21.2.0                   pypi_0    pypi
asttokens                 2.1.0                    pypi_0    pypi
attrs                     22.1.0                   pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
beautifulsoup4            4.11.1                   pypi_0    pypi
bleach                    5.0.1                    pypi_0    pypi
blosc                     1.21.1               h83bc5f7_3    conda-forge
bokeh                     3.0.1                    pypi_0    pypi
boost-cpp                 1.78.0               h75c5d50_1    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.18.1               h7f98852_0    conda-forge
ca-certificates           2022.9.24            ha878542_0    conda-forge
cairo                     1.16.0            ha61ee94_1014    conda-forge
certifi                   2022.9.24                pypi_0    pypi
cffi                      1.15.1                   pypi_0    pypi
cfitsio                   4.1.0                hd9d235c_0    conda-forge
cftime                    1.6.2                    pypi_0    pypi
click                     8.1.3                    pypi_0    pypi
click-plugins             1.1.1                    pypi_0    pypi
cligj                     0.7.2                    pypi_0    pypi
configobj                 5.0.6                    pypi_0    pypi
contourpy                 1.0.6                    pypi_0    pypi
coverage                  5.5                      pypi_0    pypi
curl                      7.86.0               h2283fc2_1    conda-forge
cycler                    0.11.0                   pypi_0    pypi
debugpy                   1.6.3                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
defusedxml                0.7.1                    pypi_0    pypi
deprecation               2.1.0                    pypi_0    pypi
entrypoints               0.4                      pypi_0    pypi
executing                 1.2.0                    pypi_0    pypi
expat                     2.5.0                h27087fc_0    conda-forge
eyed3                     0.9.7                    pypi_0    pypi
fastjsonschema            2.16.2                   pypi_0    pypi
filetype                  1.2.0                    pypi_0    pypi
fiona                     1.8.22                   pypi_0    pypi
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.14.1               hc2a2eb6_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.38.0                   pypi_0    pypi
freetype                  2.12.1               hca18f0e_0    conda-forge
freexl                    1.0.6                h166bdaf_1    conda-forge
gdal                      3.5.3                    pypi_0    pypi
geojson                   2.5.0                    pypi_0    pypi
geopandas                 0.12.1                   pypi_0    pypi
geos                      3.11.0               h27087fc_0    conda-forge
geotiff                   1.7.1                ha76d385_4    conda-forge
gettext                   0.21.1               h27087fc_0    conda-forge
giflib                    5.2.1                h36c2ea0_2    conda-forge
hdf4                      4.2.15               h9772cbc_5    conda-forge
hdf5                      1.12.2          nompi_h4df4325_100    conda-forge
icu                       70.1                 h27087fc_0    conda-forge
idna                      3.4                      pypi_0    pypi
importlib-metadata        5.0.0                    pypi_0    pypi
importlib-resources       5.10.0                   pypi_0    pypi
importlib_metadata        5.0.0                hd8ed1ab_1    conda-forge
ipykernel                 6.17.1                   pypi_0    pypi
ipython                   8.6.0                    pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
ipywidgets                8.0.2                    pypi_0    pypi
jedi                      0.18.1                   pypi_0    pypi
jinja2                    3.1.2                    pypi_0    pypi
jpeg                      9e                   h166bdaf_2    conda-forge
json-c                    0.16                 hc379101_0    conda-forge
jsonschema                4.17.0                   pypi_0    pypi
jupyter                   1.0.0                    pypi_0    pypi
jupyter-client            7.4.5                    pypi_0    pypi
jupyter-console           6.4.4                    pypi_0    pypi
jupyter-core              5.0.0                    pypi_0    pypi
jupyter-server            1.23.1                   pypi_0    pypi
jupyterlab-pygments       0.2.2                    pypi_0    pypi
jupyterlab-widgets        3.0.3                    pypi_0    pypi
kealib                    1.4.15               ha7026e8_1    conda-forge
keyutils                  1.6.1                h166bdaf_0    conda-forge
kiwisolver                1.4.4                    pypi_0    pypi
krb5                      1.19.3               h08a2579_0    conda-forge
lcms2                     2.14                 h6ed2654_0    conda-forge
ld_impl_linux-64          2.39                 hc81fddc_0    conda-forge
lerc                      4.0.0                h27087fc_0    conda-forge
libblas                   3.9.0           16_linux64_openblas    conda-forge
libcblas                  3.9.0           16_linux64_openblas    conda-forge
libcurl                   7.86.0               h2283fc2_1    conda-forge
libdap4                   3.20.6               hd7c4107_2    conda-forge
libdeflate                1.14                 h166bdaf_0    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 12.2.0              h65d4601_19    conda-forge
libgdal                   3.5.3                heb1176f_1    conda-forge
libgfortran-ng            12.2.0              h69a702a_19    conda-forge
libgfortran5              12.2.0              h337968e_19    conda-forge
libglib                   2.74.1               h606061b_1    conda-forge
libgomp                   12.2.0              h65d4601_19    conda-forge
libiconv                  1.17                 h166bdaf_0    conda-forge
libkml                    1.3.0             h37653c0_1015    conda-forge
liblapack                 3.9.0           16_linux64_openblas    conda-forge
libnetcdf                 4.8.1           nompi_h261ec11_106    conda-forge
libnghttp2                1.47.0               hff17c54_1    conda-forge
libnsl                    2.0.0                h7f98852_0    conda-forge
libopenblas               0.3.21          pthreads_h78a6416_3    conda-forge
libpng                    1.6.38               h753d276_0    conda-forge
libpq                     14.5                 he2d8382_1    conda-forge
librttopo                 1.1.0               hf730bdb_11    conda-forge
libspatialite             5.0.1               hfbd986c_21    conda-forge
libsqlite                 3.39.4               h753d276_0    conda-forge
libssh2                   1.10.0               hf14f497_3    conda-forge
libstdcxx-ng              12.2.0              h46fd767_19    conda-forge
libtiff                   4.4.0                h55922b4_4    conda-forge
libuuid                   2.32.1            h7f98852_1000    conda-forge
libwebp-base              1.2.4                h166bdaf_0    conda-forge
libxcb                    1.13              h7f98852_1004    conda-forge
libxml2                   2.10.3               h7463322_0    conda-forge
libzip                    1.9.2                hc929e4a_1    conda-forge
libzlib                   1.2.13               h166bdaf_4    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
markupsafe                2.1.1                    pypi_0    pypi
matplotlib                3.6.2                    pypi_0    pypi
matplotlib-inline         0.1.6                    pypi_0    pypi
mistune                   2.0.4                    pypi_0    pypi
munch                     2.5.0                    pypi_0    pypi
nbclassic                 0.4.8                    pypi_0    pypi
nbclient                  0.7.0                    pypi_0    pypi
nbconvert                 7.2.4                    pypi_0    pypi
nbformat                  5.7.0                    pypi_0    pypi
ncurses                   6.3                  h27087fc_1    conda-forge
nest-asyncio              1.5.6                    pypi_0    pypi
netcdf4                   1.6.1                    pypi_0    pypi
notebook                  6.5.2                    pypi_0    pypi
notebook-shim             0.2.2                    pypi_0    pypi
nspr                      4.32                 h9c3ff4c_1    conda-forge
nss                       3.78                 h2350873_0    conda-forge
numpy                     1.23.4           py38h7042d01_1    conda-forge
openjpeg                  2.5.0                h7d73246_1    conda-forge
openssl                   3.0.7                h166bdaf_0    conda-forge
packaging                 21.3                     pypi_0    pypi
pandas                    1.5.1                    pypi_0    pypi
pandocfilters             1.5.0                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pcre                      8.45                 h9c3ff4c_0    conda-forge
pcre2                     10.40                hc3806b6_0    conda-forge
pexpect                   4.8.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pillow                    9.3.0                    pypi_0    pypi
pip                       22.3.1             pyhd8ed1ab_0    conda-forge
pixman                    0.40.0               h36c2ea0_0    conda-forge
pkgutil-resolve-name      1.3.10                   pypi_0    pypi
platformdirs              2.5.3                    pypi_0    pypi
poppler                   22.11.0              h92391eb_0    conda-forge
poppler-data              0.4.11               hd8ed1ab_0    conda-forge
postgresql                14.5                 ha105346_1    conda-forge
proj                      9.1.0                h93bde94_0    conda-forge
prometheus-client         0.15.0                   pypi_0    pypi
prompt-toolkit            3.0.32                   pypi_0    pypi
psutil                    5.9.4                    pypi_0    pypi
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
ptyprocess                0.7.0                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
pycparser                 2.21                     pypi_0    pypi
pyglet                    1.1.4                    pypi_0    pypi
pygments                  2.13.0                   pypi_0    pypi
pyparsing                 3.0.9                    pypi_0    pypi
pyproj                    3.4.0                    pypi_0    pypi
pyrsistent                0.19.2                   pypi_0    pypi
python                    3.8.13          ha86cf86_0_cpython    conda-forge
python-dateutil           2.8.2                    pypi_0    pypi
python_abi                3.8                      2_cp38    conda-forge
pytz                      2022.6                   pypi_0    pypi
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     24.0.1                   pypi_0    pypi
qtconsole                 5.4.0                    pypi_0    pypi
qtpy                      2.3.0                    pypi_0    pypi
rasterio                  1.3.3                    pypi_0    pypi
readline                  8.1.2                h0f457ee_0    conda-forge
send2trash                1.8.0                    pypi_0    pypi
setuptools                65.5.1             pyhd8ed1ab_0    conda-forge
shapely                   1.8.5.post1              pypi_0    pypi
six                       1.16.0                   pypi_0    pypi
snappy                    1.1.9                hbd366e4_2    conda-forge
sniffio                   1.3.0                    pypi_0    pypi
snuggs                    1.4.7                    pypi_0    pypi
soupsieve                 2.3.2.post1              pypi_0    pypi
sqlite                    3.39.4               h4ff8645_0    conda-forge
stack-data                0.6.0                    pypi_0    pypi
terminado                 0.17.0                   pypi_0    pypi
tiledb                    2.11.3               h3f4058f_1    conda-forge
tinycss2                  1.2.1                    pypi_0    pypi
tk                        8.6.12               h27826a3_0    conda-forge
toml                      0.10.2                   pypi_0    pypi
tornado                   6.2                      pypi_0    pypi
traitlets                 5.5.0                    pypi_0    pypi
tzcode                    2022f                h166bdaf_0    conda-forge
tzdata                    2022f                h191b570_0    conda-forge
wcwidth                   0.2.5                    pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
websocket-client          1.4.2                    pypi_0    pypi
wheel                     0.38.3             pyhd8ed1ab_0    conda-forge
widgetsnbextension        4.0.3                    pypi_0    pypi
xerces-c                  3.2.4                h55805fa_1    conda-forge
xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
xorg-libice               1.0.10               h7f98852_0    conda-forge
xorg-libsm                1.2.3             hd9c2040_1000    conda-forge
xorg-libx11               1.7.2                h7f98852_0    conda-forge
xorg-libxau               1.0.9                h7f98852_0    conda-forge
xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
xorg-libxext              1.3.4                h7f98852_1    conda-forge
xorg-libxrender           0.9.10            h7f98852_1003    conda-forge
xorg-renderproto          0.11.1            h7f98852_1002    conda-forge
xorg-xextproto            7.3.0             h7f98852_1002    conda-forge
xorg-xproto               7.0.31            h7f98852_1007    conda-forge
xyzservices               2022.9.0                 pypi_0    pypi
xz                        5.2.6                h166bdaf_0    conda-forge
zipp                      3.10.0             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               h166bdaf_4    conda-forge
zstd                      1.5.2                h6239696_4    conda-forge
```
```
$ conda list proj
# packages in environment at /home/UA/tcarman2/miniconda3/envs/py38:
#
# Name                    Version                   Build  Channel
proj                      9.1.0                h93bde94_0    conda-forge
pyproj                    3.4.0                    pypi_0    pypi
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : py38
    active env location : /home/UA/tcarman2/miniconda3/envs/py38
            shell level : 2
       user config file : /home/UA/tcarman2/.condarc
 populated config files : /home/UA/tcarman2/.condarc
          conda version : 22.9.0
    conda-build version : not installed
         python version : 3.9.12.final.0
       virtual packages : __linux=3.10.0=0
                          __glibc=2.17=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/UA/tcarman2/miniconda3  (writable)
      conda av data dir : /home/UA/tcarman2/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/UA/tcarman2/miniconda3/pkgs
                          /home/UA/tcarman2/.conda/pkgs
       envs directories : /home/UA/tcarman2/miniconda3/envs
                          /home/UA/tcarman2/.conda/envs
               platform : linux-64
             user-agent : conda/22.9.0 requests/2.28.1 CPython/3.9.12 Linux/3.10.0-1160.31.1.el7.x86_64 centos/7.9.2009 glibc/2.17
                UID:GID : 17312567:16777729
             netrc file : None
           offline mode : False

```
</details>

**Comment by snowman2** [comment]
#1155

**Comment by bryevdv** [comment]
@snowman2 I'm not entirely sure this is a duplicate, or at least, there is also an issue for Bokeh to address, see: 

https://discuss.python.org/t/module-importlib-has-no-attribute-metadata/20947/12?u=bryevdv

TLDR; apparently doing `del importlib.sampledata` is very bad.

**Comment by bryevdv** [comment]
Right, in this case pyproj dist info is not missing:
```
bk301-py10 ❯ cat /Users/bryan/anaconda/envs/bk301-py10/lib/python3.10/site-packages/pyproj-3.4.0.dist-info/METADATA
Metadata-Version: 2.1
Name: pyproj
Version: 3.4.0
Summary: Python interface to PROJ (cartographic projections and coordinate transformations library)
Home-page: https://github.com/pyproj4/pyproj
Download-URL: http://python.org/pypi/pyproj
Author: Jeff Whitaker
Author-email: jeffrey.s.whitaker@noaa.gov
License: MIT
Project-URL: Documentation, https://pyproj4.github.io/pyproj/
Project-URL: Release Notes, https://pyproj4.github.io/pyproj/stable/history.html
Project-URL: Bug Tracker, https://github.com/pyproj4/pyproj/issues
Project-URL: Source Code, https://github.com/pyproj4/pyproj
Keywords: GIS,map,geospatial,coordinate-systems,coordinate-transformation,cartographic-projection,geodesic
Platform: any
Classifier: Development Status :: 4 - Beta
Classifier: Intended Audience :: Science/Research
Classifier: License :: OSI Approved :: MIT License
Classifier: Operating System :: OS Independent
Classifier: Programming Language :: Python
Classifier: Programming Language :: Python :: 3.8
Classifier: Programming Language :: Python :: 3.9
Classifier: Programming Language :: Python :: 3.10
Classifier: Programming Language :: Python :: 3.11
Classifier: Programming Language :: Python :: 3 :: Only
Classifier: Topic :: Scientific/Engineering
Classifier: Topic :: Scientific/Engineering :: GIS
Classifier: Topic :: Scientific/Engineering :: Mathematics
Classifier: Topic :: Software Development :: Libraries :: Python Modules
Classifier: Typing :: Typed
Requires-Python: >=3.8
Description-Content-Type: text/markdown
License-File: LICENSE
Requires-Dist: certifi

<snip>
```

So FWIW I would agree this can be closed, but because it is a Bokeh issue to fix, not a dupe. :)

**Comment by bryevdv** [comment]
Bokeh issue for completenes: https://github.com/bokeh/bokeh/issues/12587

**Comment by snowman2** [comment]
The issues have slightly different symptoms, but the resolution is a duplicate.
